cli.js

```diff
@@ -83,12 +83,24 @@ const handleImportDirCommand = async args => {
 }
 
 const handleGenerateFilesCommand = async args => {
-  const { dirId = 'io.cozy.files.root-dir', filesCount = 10, url, token } = args
+  const { dirId, filesCount, url, token, qualify, mime } = args
+
+  if (qualify && !token) {
+    log.warn(
+      'To view files in MyPapers, an app/kconnector token is required to create qualified files.\nThis token allows you to have the `cozyMetadata.createdByApp` prop on files.'
+    )
+  }
+
   const ach = new ACH(token || autotoken(url, ['io.cozy.files']), url, [
     'io.cozy.files'
   ])
   await ach.connect()
-  await ach.createFiles(parseInt(filesCount), dirId)
+  await ach.createFiles({
+    filesCount: parseInt(filesCount),
```

Comment:
If `filesCount`is undefined, `parseInt(undefined)` will return `NaN` so it will **not** use the default value of `createFiles` function. Potential bug?

Reply:
Tout à fait, j'ai déplacé les assignations par défaut un cran après, c'était une erreur je vais les remettre ici 👍